rustfmt.toml

```diff
@@ -0,0 +1,201 @@
+# "Use verbose output"
+verbose = false
```

Comment:
Actually, most of them are default values, they are here just for your information.